CareKitUI/CareKitUITests/TestColorExtension.swift

```diff
@@ -57,8 +56,7 @@ class TestColorExtension: XCTestCase {
 
     func testClamping() {
         let start = UIColor(red: 0, green: 0, blue: 0, alpha: 1)
-        let lightened = start.lightened(10)
+        let lightened = start.lightened(1)
```

Comment:
This has a pre-condition where the argument passed to `lightened` has to be `<=1`. For some reason it hasn't been failing on the iOS tests, but the watchOS tests caught the error.

Reply:
I found the issue. The reason was `TestColorExtension` wasn't included in the XCTest scheme so it never ran when the CI tests the Xcode project. I noticed the issue in SPM after https://github.com/carekit-apple/CareKit/pull/691#discussion_r1172662798

The update is here: https://github.com/carekit-apple/CareKit/pull/691/commits/225341b1785513785b7d4a9d4538c2dd48bc0554